client/.sample.env

```diff
@@ -3,6 +3,7 @@ CLIENT_AWS_ACCESS_KEY=access-key
 CLIENT_AWS_SECRET_KEY=access-secret
 CLIENT_AWS_BUCKET=schoolbot-assets
 CLIENT_AWS_REGION=us-east-1
+CLIENT_GOOGLE_ANALYTICS_ID=ga-app-id
```

Comment:
Let's keep these alphabetized 😃 Also, does it break anything if we just leave this blank? (indicating it's optional)